README.md

```diff
@@ -26,6 +26,9 @@ conda install -c conda-forge numpy scipy matplotlib netCDF4 xarray dask bottlene
 ```
 
 ## Running the analysis
-  1. create a configuration file by copying and modifying `config.template` or one of the example files in `configs`
-  2. run: `./run_analysis.py config.myrun`, where `config.myrun` is the config file you created
-  3. If you want to run a subset of the analysis, you can either modify the `generate` option under `[output]` in the config file or use the `--generate` flag on the command line.  See `config.template` for more details.
+  1. create a configuration file (say `config.myrun`) by copying and modifying 
+     `config.default` or one of the example files in `configs`.  Note, you only
+     need include entries that are modified from `config.default` in your 
+     confguration file; entries from `config.default` will be used by default.
```

Comment:
The procedure you describe is the right one.  Copy the machine config file first, then modify it.  Or you can even make a third config file that has your changes to both the one in `configs` and `config.default`.  Then, you would run `./run_analysis configs/lanl/config.blah.wolf config.myrun` and `config.myrun` might only need to contain the output directory you want to write to.  But that's totally up to you and maybe too complicated to explain to the typical user.